xbmc/pvr/PVRManager.cpp

```diff
@@ -793,27 +771,22 @@ void CPVRManager::OnPlaybackStarted(const CFileItemPtr item)
   {
     if (item->HasPVRChannelInfoTag())
     {
-      m_addons->SetPlayingChannel(item->GetPVRChannelInfoTag());
-    }
-    else
-    {
-      m_addons->SetPlayingRecording(item->GetPVRRecordingInfoTag());
-    }
+      const CPVRChannelPtr channel(item->GetPVRChannelInfoTag());
 
-    {
-      CSingleLock lock(m_critSection);
-      m_currentFile.reset(new CFileItem(*item));
-    }
+      m_addons->SetPlayingChannel(channel);
+
+      m_guiActions->GetChannelPreview().SetPlayingChannel(channel);
```

Comment:
CChannelPreview does a lot more than only preview. I would expect that if I have disabled preview, this class won't be required but it is used in normal code path here. Maybe find a moe appropriate name for it